app/src/main/kotlin/no/nav/k9punsj/omsorgspengermidlertidigalene/OmsorgspengerMidlertidigAleneSøknadDto.kt

```diff
@@ -11,6 +11,13 @@ import no.nav.k9punsj.objectMapper
 import java.time.LocalDate
 import java.time.LocalTime
 
+data class NyOmsMASøknad(
```

Comment:
Trenger vi den her? Eneste forskjellen her er vel att du ikke har pleietrengende? Har påbegynt jobben med att skrive om till interface så att det blir likt for alle ytelser her #557

Reply:
Eller det var søknaden i neste steg som jag har laget interface for, men poengen er att vi vill ikke ha en egen Dto for varje ytelse. Tenker det er bedre med nullable felt med kommentar isåf?

Reply:
Eller det var søknaden i neste steg som jag har laget interface for, men poengen er att vi vill ikke ha en egen Dto for varje ytelse. Tenker det er bedre med nullable felt med kommentar isåf?

Reply:
Syns ikke man skal introdusere nullable felter på en data objekt for gjenbrukbarhetens skyld. Det skaper unødvendig forvirring eller unntakshåndtering av feltene.

Reply:
Nei och jag tror ikke vi trenger det men det er så det er byggt opp nå. :)
Har ikke hunnit sette mig in i hur det brukes på tvers av ytelserne men jag er ganske sikker på att de fleste er identiska. Uansett så er det vel ikke fler identiska DTOer vi vill ha?

Reply:
Nei, vi ønsker ikke flere identiske DTOer. Da er det kanskje bedre å introdusere arv her. Trekke ut felles feltene i en super klasse, deretter lage egne DTOer med egne felter for de ytelsene som trenger de feltene.

Reply:
Ja, det er ett førsøk til det som jag begynt på i #557 :)

Reply:
Ok nice 👍 
Da foreslår jeg at denne merges inn hvis det ikke er noen andre kommentarer, slik at Vebjørn kan få utviklet videre.
Så kan vi adressere disse endringene i #557  :)

Reply:
Ja kør på, jag kan fikse refaktoreringen senere :)